vividus-to-xray-exporter/src/test/java/org/vividus/xray/databind/ManualTestCaseSerializerTests.java

```diff
@@ -56,6 +56,7 @@ void init()
     void shouldSerializeManualTest() throws IOException
     {
         ManualTestCase test = createBaseTest();
+        test.setAssignee("support@vividus.dev");
```

Comment:
should it be username or user email?

Reply:
username, but in most cases username == user email